x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.tsx

```diff
@@ -377,3 +379,8 @@ export const getActionMessageParams = memoizeOne((ruleType: Type | undefined): A
 // typed as null not undefined as the initial state for this value is null.
 export const userHasNoPermissions = (canUserCRUD: boolean | null): boolean =>
   canUserCRUD != null ? !canUserCRUD : false;
+
+export const MaxWidthEuiFlexItem = styled(EuiFlexItem)`
+  max-width: 1000px;
+  overflow: hidden;
+`;
```

Comment:
NOTE: Using this instead of the previous way as the previous way no longer works whenever the top level:

```ts
<Main ...
```

React component over the last few days has morphed into going from "display: block" to being "display: flex" and it is using a column based layout rather than previous row based. So now, we utilize this nifty `MaxWidthEuiFlexItem` here with EuiFlex in order to control our inner component and flip it back to being a row based element. This also removes all the other hand spun CSS that was centering things for the simpler Eui space around and text justifications. So much much better so far 👍